frontend/src/App.vue

```diff
@@ -1,17 +1,14 @@
 <script setup>
-import HelloWorld from "./components/HelloWorld.vue";
+import Navbar from "./components/Navbar.vue";
 </script>
 
 <template>
   <div>
-    <a href="https://vitejs.dev" target="_blank">
-      <img src="/vite.svg" class="logo" alt="Vite logo" />
-    </a>
-    <a href="https://vuejs.org/" target="_blank">
-      <img src="./assets/vue.svg" class="logo vue" alt="Vue logo" />
-    </a>
+    <h1>Title of stuff</h1>
+    <hr />
   </div>
-  <HelloWorld msg="Vite + Vue" />
+  <Navbar />
```

Comment:
lets have this navbar component styled like this https://www.airbnb.ca/
instead of adding the filter component (which we will prob be doing in the near ish future) just have it say "Filters" instead of "title of stuff"